PhysEdJournal.Api/Rest/AddPoints/Contracts/AddPointsForStandardToStudentRequest.cs

```diff
@@ -0,0 +1,40 @@
+using FluentValidation;
+using PhysEdJournal.Core.Entities.Types;
+using static PhysEdJournal.Core.Constants.ModelsConstants;
+using static PhysEdJournal.Core.Constants.PointsConstants;
+
+namespace PhysEdJournal.Api.Rest.AddPoints.Contracts;
+
+public class AddPointsForStandardToStudentRequest
+{
+    public required string StudentGuid { get; init; }
+    public required int PointsAmount { get; init; }
+    public required DateOnly Date { get; init; }
+    public required StandardType StandardType { get; init; }
+    public required bool IsOverride { get; init; }
+    public string? Comment { get; init; } = null;
+
+    public sealed class Validator : AbstractValidator<AddPointsForStandardToStudentRequest>
+    {
+        public Validator()
+        {
+            RuleFor(request => request.StudentGuid)
+                .Length(GuidLength, GuidLength)
+                .WithMessage("В поле должен передаваться гуид стандартного формата");
+            RuleFor(request => request.PointsAmount)
+                .GreaterThan(0)
+                .WithMessage("Колличество баллов должно быть больше 0")
```

Comment:
```suggestion
                .WithMessage("Количество баллов должно быть больше 0")
```

в остальных местах также